sdk/textanalytics/Azure.AI.TextAnalytics/tests/TextAnalyticsClientLiveTests.cs

```diff
@@ -12,7 +12,7 @@
 namespace Azure.AI.TextAnalytics.Tests
 {
     [ClientTestFixture(
-        TextAnalyticsClientOptions.ServiceVersion.V3_0,
+        /*TextAnalyticsClientOptions.ServiceVersion.V3_0,*/
```

Comment:
Can you expand a little bit more on why you can't do multi-version testing anymore?

Reply:
Sure. See https://github.com/Azure/azure-sdk-for-net/blob/de78a8e5b49f724d849cd09543ad37ad24283f37/sdk/textanalytics/Azure.AI.TextAnalytics/src/Generated/ServiceRestClient.cs#L49

I couldn't find a way to change this behavior and asking around, only Python is able to support multi version from generated code.
Is there a way to avoid this in .NET?

Reply:
Ah, the swagger isn't parameterized with API version, instead it's embedded directly into path.

https://github.com/Azure/azure-rest-api-specs/blob/38a485fdacf92b48427f02bc0ea3f6f8d4964614/specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.1/TextAnalytics.json#L26

What other swaggers usually do is have api-version as a parameter and you can pass it when constructing the rest client.

Reply:
Or at least what I thought they did. I'll follow up on this.